deploy/base/lookup-artifact-location-v0.1.yaml

```diff
@@ -1,59 +0,0 @@
-apiVersion: tekton.dev/v1beta1
```

Comment:
I don't think we can delete this out of the gate because of https://github.com/redhat-appstudio/infra-deployments/blob/main/components/build/jvm-build-service/kustomization.yaml#L7

I think you have to do the hardcoding of the task like you have in the operator code, but leave this file, merge this PR, and then create a infra-deployments PR that removes https://github.com/redhat-appstudio/infra-deployments/blob/main/components/build/jvm-build-service/kustomization.yaml#L7 and https://github.com/redhat-appstudio/infra-deployments/blob/main/hack/preview.sh#L98-L99

You would also want to add logic in preview.sh that updates the operator deployment to add the env var setting on the deployment based on the presence of `JVM_BUILD_SERVICE_REQPROCESSOR_IMAGE` etc.

After those merge, you can come back to this repo and delete this file.